emperor/support_files/js/controller.js

```diff
@@ -209,6 +209,9 @@ define([
         controller.resize(w, h);
       }
     });
+
+    // Set for update 
+    this.sceneViews[0].needsUpdate = true;
```

Comment:
@josenavas, yes you are right, this made me realize that in a situation with more than one `ScenePlotView3D` object, we are updating all the objects if one of them needs update, right?